linux_os/guide/system/software/integrity/fips/grub_legacy_enable_fips_mode/rule.yml

```diff
@@ -43,12 +43,18 @@ warnings:
     - general: |-
         The system needs to be rebooted for these changes to take effect.
     - regulatory: |-
-        The ability to enable FIPS does not denote FIPS compliancy or certification.
-        Red Hat, Inc. and Red Hat Enterprise Linux are respectively FIPS certified and compliant. Community
-        projects such as CentOS, Scientific Linux, etc. do not necessarily meet FIPS certification and compliancy.
-        Therefore, non-certified vendors and/or projects do not meet this requirement even if technically feasible.
+        The Federal Information Systems Modernization Act (FISMA), requires cryptography protecting sensitive
+        or valuable data to undergo FIPS 140 validation. The U.S. National Institute of Standards and
+        Technology (NIST) views unvalidated cryptography as providing no protection to the information or
+        data—in effect the data would be considered unprotected plaintext. If the agency specifies that the
+        information or data be cryptographically protected, FIPS 140 is applicable. This configuration
+        check will fail on platforms lacking FIPS 140 validation, such as the CentOS, Scientific Linux,
+        and Fedora projects, even if FIPS-approved ciphers can be installed and enabled.
         <br /><br />
-        See <b>{{{ weblink(link="http://csrc.nist.gov/groups/STM/cmvp/documents/140-1/1401vend.htm") }}}</b>
-        for a list of FIPS certified vendors.
-
-platform: machine
```

Comment:
`platform: machine` shouldn't be removed here.

Reply:
It looks like it is set in the group.yml so I would say it is safe to remove.
https://github.com/ComplianceAsCode/content/blob/d9fbc2ad6e506e7fc8eeb78fb2d3fffc47333e41/linux_os/guide/system/software/integrity/fips/group.yml#L18

Reply:
@ggbecker  Nice, it should be fine then!